docs/metrics.md

```diff
@@ -76,20 +76,32 @@ All memory pool metrics have the following tags:
 | `runtime.jvm.threads.peak`   | The peak live thread count since the Java virtual machine started or peak was reset.
 | `runtime.jvm.threads.states` | The current number of threads per `state` (metric tag).
 
-### Apache DBCP2 connection pool
+### Connection pool metrics
 
-| Metric name                    | Description |
-| ------------------------------ | ----------- |
-| `db.pool.connections`          | The number of open connections.
-| `db.pool.connections.active`   | The number of open connections that are currently in use.
-| `db.pool.connections.idle`     | The number of open connections that are currently idle.
-| `db.pool.connections.idle.max` | The maximum number of idle open connections allowed.
-| `db.pool.connections.idle.min` | The minimum number of idle open connections allowed.
-| `db.pool.connections.max`      | The maximum number of open connections allowed.
+Splunk Distribution of OpenTelemetry Java instruments several JDBC connection pool implementations:
+
+* [Apache DBCP2](https://commons.apache.org/proper/commons-dbcp/)
+* [HikariCP](https://github.com/brettwooldridge/HikariCP)
+
+Each of the supported connection pools reports a subset of the following metrics:
+
+| Metric name                           | Description |
+| ------------------------------------- | ----------- |
+| `db.pool.connections`                 | The number of open connections.
+| `db.pool.connections.active`          | The number of open connections that are currently in use.
+| `db.pool.connections.idle`            | The number of open connections that are currently idle.
+| `db.pool.connections.idle.max`        | The maximum number of idle open connections allowed.
```

Comment:
since hikari doesn't support this, should we add a comment about that here?

Reply:
To be honest I'm still wondering how to handle docs for those connection pools - I've started looking into another one (`tomcat-jdbc`) and it seems that it supports yet another subset of these metrics (and maybe a few new ones, who knows). I think there are 2 or 3 more connection pools to cover - after I'm done with instrumenting them I'll probably reserve some time for making them a bit more uniform, and for fixing those docs too.
Anyway, I'm thinking of either adding a column with a list of connection pools that support these metrics or having a separate table for each pool implementation.